specs/experimental/validator-system-v2/validator-manager.md

```diff
@@ -0,0 +1,173 @@
+| `NONE`       | X         | X         | X                     | X                     |
+| `EXITED`     | O         | O/X       | X                     | O/X                   |
+| `REGISTERED` | O         | X         | O                     | X                     |
+| `READY`      | O         | X         | O                     | O                     |
+| `INACTIVE`   | O         | O         | O                     | X                     |
+| `ACTIVE`     | O         | O         | O                     | O                     |
+
+Only validators with an `ACTIVE` status can submit outputs.
+
+### Jail
+
+If a validator has not submitted output when it was selected as a priority validator, the validator's
+`noSubmissionCount` is incremented by 1. If the validator submits output the next time it is selected as a priority
+validator, `noSubmissionCount` is reset to 0. Otherwise, if the `noSubmissionCount` exceeds `JAIL_THRESHOLD`, the
+validator is deemed to have failed to fulfill its obligations. Accordingly, the validator is sent to jail and removed
+from the validator tree. After `JAIL_PERIOD` has elapsed, the validator can be unjailed by calling `tryUnjail` function
+on its own and activated if the activation conditions are met.
+
+## Entry Point
+
+Validator Manager contract acts as an entry point for external contracts to slash challenge losers or distribute output
+rewards. Below is a detailed description of each case.
+
+### Slashing
+
+When the challenge ends, the challenge loser is slashed. To apply the slash to the loser, the Colosseum contract calls
+Validator Manager, which calls Asset Manager to calculate the slash amount and apply the change. The loser is sent to
+jail and the slash amount is accumulated as a pending challenge reward.
+
+### Reward Distribution
+
+Each time a submitted output is finalized, the reward for submitting the output is distributed to the output submitter.
+As mentioned in [Reward distribution](./overview.md#reward-distribution), each reward is calculated as follows:
+
+| Rewards For Validators                                             | Rewards For KRO Delegators          | Rewards For KGH Delegators             |
+|--------------------------------------------------------------------|-------------------------------------|----------------------------------------|
+| $\verb#BASE_REWARD# \times c_i + \verb#BOOSTED_REWARD# \times c_i$ | $\verb#BASE_REWARD# \times (1-c_i)$ | $\verb#BOOSTED_REWARD# \times (1-c_i)$ |
+
+Here, $c_i$ is the commission rate the validator set during the validator registration process. `BOOSTED_REWARD` is
+calculated based on the number of delegated KGH, the exact expression is $$8 \times arctan(0.01 \times G_i)$$, $G_i$
+denotes the number of KGH delegated to $i^{th}$ validator. This is designed to have the effect that the boosted reward
+is constantly decreasing while the number of KGH increases.
```

Comment:
It would be great if we could add a link to our docs for more details: https://docs.kroma.network/kroma-guardian-house/kgh-utilities#boosting-validator-reward.